torchrl/objectives/a2c.py

```diff
@@ -234,6 +250,7 @@ def __init__(
         functional: bool = True,
         actor: ProbabilisticTensorDictSequential = None,
         critic: ProbabilisticTensorDictSequential = None,
+        return_tensorclass: bool = False,
```

Comment:
Should be added to the docstrings

Reply:
working on it.

Reply:
@vmoens added doctests for tensorclass changes. but I see some doctest issues and blockers. can you please help me resolve. 

- there are some existing doctest failures, we might need a separate task to address. 
- what would be the aggregate_loss for each loss within tensorclass? 
- there are some existing errors like 
> 1.        ```Cannot interpret 'torch.int64' as a data type```
> 2.        ```'key "action_value" not found in TensorDict with keys [\'done\', \'logits\', \'observation\', \'reward\', \'state_value\', \'terminated\']' ```
> 3.        ```NameError: name 'actor' is not defined```